hspy_bundle/NSIS_installer_script.nsi

```diff
@@ -307,7 +307,7 @@ SectionIn RO
 	File /r "${WINPYTHON_PATH}\*"
 	${If} $InstMode = 2
 	; Create right-click context menu entries for Hyperspy Here
-		Exec 'cmd.exe /C ""${APP_INSTDIR}\WinPython Command Prompt.exe" install_hyperspy_here & exit"'
+		Exec 'cmd.exe /C ""${APP_INSTDIR}\WinPython Command Prompt.exe" jupyter-context_menu-add & exit"'

```

Comment:
Given that the outcome will be similar, I'll rather live it as it is
for this release and we'll experiment with this for a future release if
that's ok.


On Thu, 2016-02-18 at 03:15 -0800, Vidar Tonaas Fauske wrote:
> In hspy_bundle/NSIS_installer_script.nsi:
> > @@ -307,7 +307,7 @@ SectionIn RO
> >  	File /r "${WINPYTHON_PATH}\*"
> >  	${If} $InstMode = 2
> >  	; Create right-click context menu entries for Hyperspy
> Here
> > -		Exec 'cmd.exe /C ""${APP_INSTDIR}\WinPython
> Command Prompt.exe" install_hyperspy_here & exit"'
> > +		Exec 'cmd.exe /C ""${APP_INSTDIR}\WinPython
> Command Prompt.exe" jupyter-context_menu-add & exit"'
> I would replace all patterns of
> Exec 'cmd.exe /C ...\WinPython Command Prompt.exe" ... & exit"'
> Sleep ...
> with
> ExecWait '"${APP_INSTDIR}\scripts\env.bat & ..."
> Haven't tested extensively, but it should work better than the
> current one which leaves the cmd window open after install.
> —
> Reply to this email directly or view it on GitHub.